src/transformStateWithClones.js

```diff
@@ -64,8 +64,35 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, transforms) {
-  // write code here
+  let tmpState = { ...state };
+  const states = [];
```

Comment:
```suggestion
  const tmpState = { ...state };
  const states = [];
```
You may use `const` for object link. Even if you will change the object, the link still will be constatn